### PR TITLE
Admin Router Service Dashboards

### PR DESCRIPTION
**Pull request by timaa2k**
This PR adds a number of things we have been working on for Admin Router dashboards:

1. [DCOS_OSS-4972](https://jira.mesosphere.com/browse/DCOS_OSS-4972) Fix Admin Router dashboard linking for previously existing dashboards/graphs
2. [DCOS-51189](https://jira.mesosphere.com/browse/DCOS-51189) Count access denied (401 Unauthorized) on the server level
3. [DCOS_OSS-4665](https://jira.mesosphere.com/browse/DCOS_OSS-4665) Add latency histogram data graphs for all requests 
4. [DCOS_OSS-5015](https://jira.mesosphere.com/browse/DCOS_OSS-5015) Add dedicated dashboards for DC/OS services accessed through Admin Router

### CODE REVIEW

**Review by DominikDary — APPROVED** [review]

File: dashboards/AdminRouter/AdminRouter-Details-Runtime.json [hunk 1]
```diff
@@ -1075,7 +1068,7 @@
     ]
   },
   "timezone": "",
-  "title": "Admin Router: Details Runtime",
+  "title": "/AdminRouter/AdminRouter-Details-Runtime",
```
Comment (gracedo) [inline dashboards/AdminRouter/AdminRouter-Details-Runtime.json]: Looks like all the dashboard titles got reverted, can we change them back to the old format? Thanks!

**Review by gracedo — CHANGES_REQUESTED** [review]

File: dashboards/AdminRouter/AdminRouter-Details-Runtime.json (hunk 1 above)
Comment (timaa2k) [reply dashboards/AdminRouter/AdminRouter-Details-Runtime.json]: Whenever the dashboards specified in the beta-dcos-monitoring package are automatically imported, the titles get converted.
How do I prevent that? It also fks up linking of graphs every time^^

Comment (gracedo) [reply dashboards/AdminRouter/AdminRouter-Details-Runtime.json]: This has been fixed in dcos-monitoring, but it won't be released till the GA release, so unfortunately the beta will continue to do this :(

**Comment by timaa2k** [comment]
@gracedo fixed titles and a lot of other things :)

**Review by gracedo — APPROVED** [review]

**Pull request merged**